src/lib/components/DashNgl.react.js

```diff
@@ -0,0 +1,228 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import {Stage, Selection} from 'ngl';
+
+/**
+ * The Molecule3dViewer is used to render schematic diagrams
```

Comment:
```suggestion
 * The NglViewer is used to render schematic diagrams
```

Could you also please rename the component to just `Ngl` or `NglMoleculeViewer` (or something else along those lines)? That would be more in line with the rest of our components.

Reply:
Ngl Viewer is already take by the developers of Ngl
http://nglviewer.org/

I am not sure if I am allowed to do that since this might lead to confusion between this Dash component and their Jupyter Widget maybe NglMoleculeViewer is better ?

Reply:
Yes, NglMoleculeViewer sounds good to me!

Reply:
The component should only be used in the Dash Bio namespace regardless, so I don't imagine there will be much confusion either way - but better to be safe than sorry :)

Reply:
Ok so does this involve renaming all the filenames as well or only inside DashNgl.react.js

Reply:
I renamed all the files

Reply:
Please just rename the `DashNgl.react.js` file to `NglMoleculeViewer.react.js`, and re-run the build script - it should auto-update everything :)

Reply:
Done :)